dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,338 @@
+package dbfit.environment;
+
+import dbfit.annotations.DatabaseEnvironment;
+import dbfit.api.AbstractDbEnvironment;
+import dbfit.fixture.StatementExecution;
+import dbfit.util.DbParameterAccessor;
+import dbfit.util.DdlStatementExecution;
+import dbfit.util.Direction;
+import dbfit.util.NameNormaliser;
+import dbfit.util.Options;
+import dbfit.util.TypeNormaliserFactory;
+import fit.TypeAdapter;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+
+@DatabaseEnvironment(name="Informix", driver="com.informix.jdbc.IfxDriver")
+public class InformixEnvironment extends AbstractDbEnvironment  {
+
+    @Override
+    public void afterConnectionEstablished() throws SQLException {
+        if (currentConnection.getMetaData().supportsTransactions()) {
+            Options.setOption(Options.OPTION_AUTO_COMMIT, "false");
+            currentConnection.setAutoCommit(false);
+        } else {
+            Options.setOption(Options.OPTION_AUTO_COMMIT, "true");
+            currentConnection.setAutoCommit(true);
+        }
+    }
+
+    @Override
+    public void connect(String connectionString, Properties info) throws SQLException {
```

Comment:
Do we need to override that method? (Looks pretty similar to the one in the base class except we don't register the driver here).